core/templates/dev/head/exploration_editor/settings_tab/settings_tab.html

```diff
@@ -8,13 +8,19 @@
             <div class="form-group" ng-class="{'has-error': !explorationTitleService.displayed}">
               <label for="explorationTitle" class="col-lg-2 col-md-2 col-sm-2">Title</label>
               <div class="col-lg-10 col-md-10 col-sm-10">
-                <input id="explorationTitle" type="text" class="form-control protractor-test-exploration-title-input" ng-model="explorationTitleService.displayed" ng-blur="saveExplorationTitle()" placeholder="Choose a title for your exploration." focus-on="<[EXPLORATION_TITLE_INPUT_FOCUS_LABEL]>">
+                <input id="explorationTitle" type="text" class="form-control protractor-test-exploration-title-input" ng-model="explorationTitleService.displayed" ng-blur="saveExplorationTitle()" placeholder="Choose a title for your exploration." focus-on="<[EXPLORATION_TITLE_INPUT_FOCUS_LABEL]>" maxlength="40">
+                <span class="help-block" style="font-size: smaller">
```

Comment:
maybe make this red like the other warning? probably better to add to the css class so it's consistent.